packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -31,586 +32,508 @@ const behaviorMenu: ComponentMenuItem[] = require('../../behaviorMenu');
+          }}
+        >
+          Fluent <span style={gradientTextStyles}>UI</span>
+        </Text>
+        <CopyToClipboard value={`yarn add ${pkg.name}@${pkg.version}`} timeout={3000}>
+          {(active, onClick) => (
+            <Box
+              as="code"
+              onClick={onClick}
+              styles={{
+                display: 'block',
+                fontWeight: 'normal',
+                fontSize: '12px',
+                opacity: active ? 1 : 0.6,
+                color: active ? 'rgb(138, 255, 124)' : 'inherit',
+                marginTop: '10px',
+                cursor: 'pointer',
+                ...(!active && {
+                  ':hover': {
+                    opacity: 0.75,
+                  },
+                }),
+              }}
+            >
+              {active ? 'Copied! Happy coding :)' : `yarn add ${pkg.name}@${pkg.version}`}
             </Box>
-          </a>
-          <Input
-            styles={topItemTheme}
-            inverted
-            fluid
-            clearable
-            icon={<SearchIcon />}
-            placeholder="Search"
-            iconPosition="end"
-            role="search"
-            onChange={this.handleQueryChange}
-            value={this.state.query}
-            inputRef={this.searchInputRef}
-          />
-        </Flex>
-        <HierarchicalTree
-          items={allSections}
-          renderItemTitle={titleRenderer}
-          activeIndex={this.state.activeCategoryIndex}
-          onActiveIndexChange={this.treeActiveIndexChanged}
+          )}
+        </CopyToClipboard>
+      </Flex>
+      <Flex column>
+        <a href={config.repoURL} target="_blank" rel="noopener noreferrer" style={topItemTheme}>
+          <Box>
+            GitHub
+            <Image src="public/images/github.png" width="20px" height="20px" styles={{ float: 'right' }} />
+          </Box>
+        </a>
+        <NavLink to="/builder" exact style={topItemTheme} activeStyle={{ fontWeight: 'bold' }}>
+          <Box>
+            Builder
+            <span style={{ border: 'orange', color: 'orange', marginLeft: '0.5rem' }}>alpha</span>
+            <EditIcon styles={{ float: 'right' }} />
+          </Box>
+        </NavLink>
+        <a href={changeLogUrl} target="_blank" rel="noopener noreferrer" style={topItemTheme}>
+          <Box>
+            CHANGELOG
+            <FilesTxtIcon styles={{ float: 'right' }} />
+          </Box>
+        </a>
+        <Input
+          styles={topItemTheme}
+          inverted
+          fluid
+          clearable
+          autoComplete="off"
+          icon={<SearchIcon />}
+          placeholder="Search"
+          iconPosition="end"
+          role="search"
+          onChange={(_, { value }) => {
+            setQuery(value);
+          }}
+          value={query}
+          inputRef={searchInputRef}
         />
-      </Segment>
-    );
-  }
-}
+      </Flex>
+      <Tree
+        items={allSections}
+        renderItemTitle={SidebarTitle}
```

Comment:
```suggestion
        renderItemTitle={renderSidebarTitle}
```

As it's not a component, it's a function

Reply:
renamed